.github/workflows/web-ui-automation.yaml

```diff
@@ -121,33 +121,4 @@ jobs:
       with:
         find: ${{ steps.previous_shasum.outputs.SHASUM }}
         replace: ${{ needs.build_and_release.outputs.new_sha }}
-        include: "prysm_repo/WORKSPACE" 
-
-    - name: Setup bazel
-      uses: jwlawson/actions-setup-bazel@v1
-      with:
-        bazel-version: '5.4.0'
-
-    - name: Run Bazel
-      run: cd prysm_repo && bazel build //validator/web:site_data
-
-    - name: Copy build site data
-      run: cd prysm_repo && cp bazel-bin/validator/web/site_data.go ./validator/web/site_data.go
-
-    - name: Run go FMT
-      run: cd prysm_repo && gofmt -w -s ./validator/web/site_data.go
-
-    - name: Create Pull Request
-      uses: peter-evans/create-pull-request@v3
-      with:
-        commit-message: Web-ui-version-${{ steps.current_version.outputs.TAG }}-commit
-        title: 'Prysm Web UI Release ${{ steps.current_version.outputs.TAG }}'
-        body: >
-          This PR is auto-generated by 
-          [create-pull-request](https://github.com/peter-evans/create-pull-request).
-        labels: automated pr
-        path: prysm_repo
-        #reviewers: rauljordan, james-prysm
-        token: ${{ secrets.PAToken }}
-        branch: prysm-web-ui-${{ steps.current_version.outputs.TAG }}
-        base: develop
\ No newline at end of file
+        include: "prysm_repo/WORKSPACE" 
```

Comment:
I think the whole task for `prysm_pull_request` can be deleted.

Reply:
oops ok looks like raul merged this one, can make another later.